flink-streaming-java/src/main/java/org/apache/flink/streaming/experimental/CollectSink.java

```diff
@@ -28,8 +29,12 @@
 import java.net.Socket;
 
 /**
- * A specialized data sink to be used by DataStreamUtils.collect.
+ * A specialized data sink to be used by DataStreamUtils.collect().
+ *
+ * <p>This experimental class is relocated from flink-streaming-contrib. Please see package-info.java
+ * for more information.
  */
+@PublicEvolving
```

Comment:
Would this better be `@Internal`? I think it is not meant to be instantiated directly, but only via the collect() call...

Reply:
make sense. Changed